src/request.ts

```diff
@@ -5,19 +5,37 @@ import 'isomorphic-fetch';
  * Return a promise that is resolved or rejected depending on the response's
  * status code.
  */
-export const checkStatus = (res: Response): Promise<Response> => {
+const _checkStatus = (res: Response): Promise<Response> => {
     if (res.status >= 300) {
```

Comment:
why 300s? with our api implementation this shouldnt be an issue, but why not reject >= 400's

Reply:
That's a good call. It's what the code internally was doing, but I agree it should be `>= 400`. A 301 redirect for instance shouldn't trigger this